### PR TITLE
Kafka quick start generate projects with maven plugin

### PR DESCRIPTION
**Pull request by ozangunalp**
Use quarkus-maven-plugin:create with noCode option to create projects without generated resources and tests.

### CODE REVIEW

File: docs/src/main/asciidoc/kafka-reactive-getting-started.adoc [hunk 1]
```diff
@@ -523,16 +534,16 @@ First, compile both applications as native:
...
 ----
 
 Run the system with:
 
 [source, bash]
 ----
-> export QUARKUS_MODE=native
-> docker compose up
+export QUARKUS_MODE=native
+docker compose up
```
Comment (cescoffier) [inline docs/src/main/asciidoc/kafka-reactive-getting-started.adoc:546]: What about adding --build ?

**Review by cescoffier — APPROVED** [review]
The QuickStart code will need to be updated

File: docs/src/main/asciidoc/kafka-reactive-getting-started.adoc (hunk 1 above)
Comment (ozangunalp) [reply docs/src/main/asciidoc/kafka-reactive-getting-started.adoc:546]: Oh forgot about that

**Comment by geoand** [comment]
General question not really tied to this PR: Should we not be pointing users to the Quarkus CLI for creating projects instead of the Maven goal?

**Comment by cescoffier** [comment]
So, the question is: do we expect the users to have downloaded the CLI first? 

At some point, we will need a multi-tabs widget providing all the solutions (cli, maven, gradle, code.quarkus.io). Easy to do if you have CSS skills. 

The same will also apply when to mention the added dependency (maven, gradle groovy, gradle kotlin)

**Review by cescoffier — APPROVED** [review]

**Comment by ozangunalp** [comment]
> At some point, we will need a multi-tabs widget providing all the solutions (cli, maven, gradle, code.quarkus.io). Easy to do if you have CSS skills. 

:+1: You beat me to this with time difference :)

**Pull request merged**